### ISSUE TITLE
broken, unresolvable shlib `libgdbm.so.4'

### ISSUE THREAD

**Issue by optik-aper** [voidlinux/xbps#279]
Two different machines running void started doing this today when running `xbps-install -Su`.

```
[*] Updating `https://repo.voidlinux.eu/current/musl/armv6l-musl-repodata' ...
armv6l-musl-repodata: 1237KB [avg rate: 993KB/s]
lighttpd-1.4.49_1: broken, unresolvable shlib `libgdbm.so.4'
zsh-5.4.2_1: broken, unresolvable shlib `libgdbm.so.4'
Transaction aborted due to unresolved shlibs.
```

**Comment by maxice8** [comment]
Should be fixed, gdbm update to 1.14 took some time.

**Comment by optik-aper** [comment]
All is well.  Thanks!
